src/content/releases/_index.en.md

```diff
@@ -5,6 +5,26 @@ pre = "<b>5. </b>"
 weight = 25
 +++
 
+## v0.5.0 Lighthouse service discovery alignment
+
+> This release mainly focused on continuing the alignment of Lighthouse's service discovery support with the [Kubernetes Multicluster
+> Services KEP][MCS KEP].
+
+* **Lighthouse** has been modified per the [Kubernetes Multicluster Services KEP][MCS KEP] as follows:
+  - The `MultiClusterService` resource has been replaced by `ServiceImport`.
+  - The `ServiceExport` resource is now updated with status information as lifecycle events occur.
+* **Lighthouse** now allows a `ServiceExport` resource to be created prior to the associated `Service`.
+* Network discovery was moved from `subctl` to the subamriner operator.
+* Several new commands were added to `subctl`: `export service`, `show versions`, `show connections`, `show networks`,
```

Comment:
Might worth mention that 'show networks' is replacing the old command 'subctl info'.
Another new command is 'show endpoints'